js/index.js

```diff
@@ -38,5 +38,76 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+// Images
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let cta = document.getElementById("cta-img");
+cta.setAttribute('src', siteContent["cta"]["img-src"])
+
+let mainContent = document.getElementById("middle-img");
+mainContent.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+//Navs
+let nav = document.querySelector("nav");
+
+let navs = document.querySelectorAll("a");
+navs.forEach((index, i) => navs[i].textContent = siteContent["nav"][`${
+  Object.keys(siteContent["nav"])[i]
+}`]);
+
+
+
+///CTA 
+let ctaH1 = document.querySelector('h1');
+ctaH1.innerHTML = siteContent['cta']['h1'].split(' ').join(`<br>`);
+
+let ctaButton = document.querySelector('button');
+ctaButton.textContent = siteContent['cta']['button'];
+
+
+
+///main-content
+let hFour = document.querySelectorAll('h4');
+
+let mainHFour = Object.keys(siteContent['main-content']).filter((key) => key.includes('h4'));
+
+hFour.forEach((index, i) => hFour[i].textContent = siteContent['main-content'][`${mainHFour[i]}`]);
```

Comment:
nicely done. I would just be careful with your naming conventions here. Your second argument in your forEach callback, the `i`, actually refers to the index. Calling your first argument index kind of just muddies the water.